test/relations.test.js

```diff
@@ -3790,7 +3790,7 @@ describe('relations', function() {
           return Person.findById(personId);
         })
         .then(function(person) {
-          person.passportItem().toObject().should.eql({name: 'Jim'});
+          person.passportItem().toObject().should.eql({name: 'Mitsos'});
```

Comment:
Hi, I just wanted to note for the test. The correct operation would be indeed to see `Jim` which was implemented in my pull request, since you alter the created embedded object that is returned by the create of passport Item and then save it.

This functionality was implemented by adding `EmbedsOne.prototype.prepareEmbeddedInstance` function that calls the parents update when embedded model save requested in a similar way that EmbedsMany relations work.
[See](https://github.com/strongloop/loopback-datasource-juggler/pull/949/files#diff-20531c2f86840f4a537a4db9f742b0e2R2084)